protostar/testing/test_scheduler.py

```diff
@@ -2,16 +2,60 @@
 import signal
 from pathlib import Path
 from typing import TYPE_CHECKING, Callable, List, Optional
+import dataclasses
+
+from protostar.testing import (
+    TestingSummary,
+)
+from protostar.testing import (
+    BrokenTestSuiteResult,
+    TestResult,
+)
 
 from .test_collector import TestCollector
 from .test_runner import TestRunner
 from .test_shared_tests_state import SharedTestsState
 from .testing_seed import Seed
 
+
 if TYPE_CHECKING:
     from protostar.commands.test.testing_live_logger import TestingLiveLogger
 
 
+def make_path_relative_if_possible(test_result: TestResult, path: Path) -> TestResult:
+    try:
+        test_result = dataclasses.replace(
+            test_result,
+            file_path=test_result.file_path.resolve().relative_to(path.resolve()),
+        )
+    except ValueError:
+        # We do this to preserve the functionality of running tests that are outside of the project
+        pass
+    return test_result
+
+
+def _update_summary(
```

Comment:
Is this used?

Reply:
Apparently no, I must've forgotten to remove it, sorry

Reply:
[format + names + remove redundant function](https://github.com/software-mansion/protostar/pull/1254/commits/f62b551533db9762d7ce549c5fa2823cc8965a14)